build-scripts/build-omm-with-icons.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+usage="
+This script builds ommpfritt with icons.
+Please give the path to ommpfritt source as an argument.
+"
+
+if [ ! "$1" ]; then
+    echo "$usage"
+    exit 1
+fi
+
+set -e
+
+source=$1
+shift
+config=${1:-Release}
```

Comment:
If the configuration argument is not set, but additional arguments are passed to cmake  (`"$@"` in line 31), this shift will eat the first of those arguments.

Proper solution would be named arguments (e.g. `--config Release`) or non-optional argument .